testing/conftest.py

```diff
@@ -15,49 +15,56 @@ def base_url(httpserver):
 
 @pytest.fixture
 def httpserver_base_url(httpserver):
-    return '--base-url={0}'.format(base_url(httpserver))
+    return "--base-url={0}".format(base_url(httpserver))
 
 
 @pytest.fixture(autouse=True)
 def testdir(request, httpserver_base_url):
     item = request.node
-    if 'testdir' not in item.funcargnames:
+    if "testdir" not in item.funcargnames:
         return
 
-    testdir = request.getfixturevalue('testdir')
+    testdir = request.getfixturevalue("testdir")
 
-    testdir.makepyfile(conftest="""
+    testdir.makepyfile(
+        conftest="""
         import pytest
         @pytest.fixture
         def webtext(base_url, selenium):
             selenium.get(base_url)
             return selenium.find_element_by_tag_name('h1').text
-        """)
+        """
+    )
 
-    testdir.makefile('.cfg', setup="""
+    testdir.makefile(
+        ".cfg",
+        setup="""
         [tool:pytest]
         filterwarnings =
             error::DeprecationWarning
             ignore:--firefox-\w+ has been deprecated:DeprecationWarning
-            ignore:MarkInfo:DeprecationWarning:pytest_selenium.drivers.firefox:88
-    """)
+            ignore:MarkInfo:DeprecationWarning:pytest_selenium.drivers.firefox
```

Comment:
I'm okay to approve this here, but I think we shouldn't ignore these warnings. Could you follow up with a patch that changes the `firefox_options` fixture to not use the deprecated `get_marker`?

Reply:
Will do!